### ISSUE TITLE
Empty table header accessibility error when using tabular data

### ISSUE THREAD

**Issue by starr027-umn** [ckeditor/ckeditor5#19204]
## 📝 Provide detailed reproduction steps (if any)

1. Create a table with both column and row headers
2. Inspect the upper left cell and note that the empty cell shows up as a th

### ✔️ Expected result

When creating a table like this, the th cell in the upper left flags as an accessibility error.  In these cases it should be a td

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
